.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.env
 
 npm-debug.log*
 yarn-debug.log*
```

Comment:
- [ ] As mentioned by the last reviewer, all the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted. Please, delete `logo192.png`, `logo512.png`, `favicon.ico` from your repo. 
- [ ] Also, since you've already committed the unwanted `.env`, after adding it to `.gitignore`, you still need to manually delete it from Github because it's still visible.